src/Accelerometers/Accelerometers.cpp

```diff
@@ -479,12 +477,12 @@ GCodeResult Accelerometers::StartAccelerometer(GCodeBuffer& gb, const StringRef&
 # if SUPPORT_CAN_EXPANSION
 		if (device.IsRemote())
 		{
-			reprap.GetExpansion().AddAccelerometerRun(device.boardAddress, 0);
+			reprap.GetExpansion().AddAccelerometerRun(device.boardAddress, 0, {0});
 		}
 		else
 # endif
 		{
-			AddLocalAccelerometerRun(0);
+			AddLocalAccelerometerRun(0, {0});
```

Comment:
Need to pass an array of the correct size here because the compiler doesn't know how many values are expected